tools/mkmapdata/mkmapdata.sh

```diff
@@ -378,7 +378,6 @@ case $hostname in
   #intelvers=12.1.5
   module purge
   module load intel/$intelvers
-  module load ncl
```

Comment:
Changes here follow this [conversation.](https://github.com/ESCOMP/CTSM/pull/823#discussion_r385317926)